### DISCUSSION TITLE
Is there a possibility to get code coverage with this service?

### DISCUSSION THREAD

**Discussion by m4hdyar** [webdriverio-community/wdio-electron-service#66] [Q&A]
I would like to know if there is any possibility of capturing code coverage. I tried using the DevTools service along with this service, but I was unsuccessful. I am curious if anyone has experience with capturing code coverage and could offer advice.

**Comment by goosewobbler** [comment]
I haven't tried capturing code coverage in WDIO yet.  In what way was the DevTools service unsuccessful when run with this service?  

@christian-bromann is there any reason why the DevTools service wouldn't work in an Electron context?

**Reply by christian-bromann** [reply]
In theory I believe it's both Chromium just that the Electron version might run a lower Chromium version than running the browser. That said I am sure there might be differences, so hard to say.
